config/environments/development.rb

```diff
@@ -60,4 +60,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  ENV['FACEBOOK_APP_ID'] = '229696334651787';
```

Comment:
It's preferable to store this in an `.env` file which you don't commit to a public repo. You can use a variable environment manager like `figaro` or `dotenv` to handle this. Nevertheless, this makes it smooth for people to test your app quickly.

When deploying to Heroku though, I'd suggest getting some practice with `figaro` or `dotenv` using your environment variables this way.

[OPTIONAL]